### PR TITLE
GH-203

### PR DESCRIPTION
**Pull request by evdhoudt**
Closes NCI-Agency/anet#203

### CODE REVIEW

File: src/main/java/mil/dds/anet/beans/Person.java [hunk 1]
```diff
@@ -122,23 +122,23 @@ public String getRank() {
...
 	}
 
 	public String getDomainUsername() {
 		return domainUsername;
 	}
 
 	public void setDomainUsername(String domainUsername) {
-		this.domainUsername = domainUsername;
+		this.domainUsername = Utils.trimStringReturnNull(domainUsername);
 	}
```
Comment (maradragan) [inline src/main/java/mil/dds/anet/beans/Person.java]: I don't think domainUsername should be trimmed, as this one comes from AD ... I think.

File: src/main/java/mil/dds/anet/beans/Person.java [hunk 2]
```diff
@@ -78,35 +78,35 @@ public void setPendingVerification(Boolean pendingVerification) {
...
 
 	public String getEmailAddress() {
-		return emailAddress;
+		return Utils.trimStringReturnNull(emailAddress);
 	}
```
Comment (maradragan) [inline src/main/java/mil/dds/anet/beans/Person.java]: Triming should only be done on set methods.

File: client/src/components/FormField.js [hunk 3]
```diff
@@ -315,6 +315,24 @@ export default class FormField extends Component {
...
+		let formContext = this.context.formFor
+		if (formContext)
+			formContext[id] = value
+		return formContext
+	}
+
+	sanitizeInput(value) {
+		if (typeof value === 'string' || value instanceof String) {
+			return value.trim()
+		}
```
Comment (maradragan) [inline client/src/components/FormField.js]: I think you should exempt here any AD related fields, like domainUsername.

File: client/src/components/FormField.js [hunk 4]
```diff
@@ -184,7 +184,7 @@ export default class FormField extends Component {
 		}
 
 		return (
-			<FormGroup controlId={id} className={className} validationState={validationState}>
+			<FormGroup controlId={id} className={className} validationState={validationState} onBlur={this.onBlur}>
```
Comment (maradragan) [inline client/src/components/FormField.js]: I don't see where this.onBlur comes from.

File: client/src/components/FormField.js [hunk 5]
```diff
@@ -184,7 +184,7 @@ export default class FormField extends Component {
 		}
 
 		return (
-			<FormGroup controlId={id} className={className} validationState={validationState}>
+			<FormGroup controlId={id} className={className} validationState={validationState} >
```
Comment (gjvoosten) [inline client/src/components/FormField.js]: You might want to remove the space before the '>'.

**Pull request merged**